rpcserver.go

```diff
@@ -4121,6 +4122,10 @@ func createRPCOpenChannel(r *rpcServer, dbChannel *channeldb.OpenChannel,
 
 	// Fetch the set of aliases for the channel.
 	channelAliases := r.server.aliasMgr.GetAliases(dbScid)
+	peerScidAlias, err := r.server.aliasMgr.GetPeerAlias(chanID)
```

Comment:
This will return error `no peer alias found` in the case where the alias scid feature isn't used, thus making `ListChannel` fail (and potenitally `openChannel`) if you  have a channel without the alias scid feature.

You may want to either check if the error being returned is `errNoPeerAlias` or check whether the alias scid feature is being used for the channel.

Reply:
Good catch. I'll fix this.

Reply:
Fixed.